### DISCUSSION TITLE
What is Docket image for pocketbase 0.6 for heroku

### DISCUSSION THREAD

**Discussion by triborg34** [pocketbase/pocketbase#517] [Q&A] [answered]
Hi i am using Pocketbase 0.6
and i want to deploy my pocketbase on heroku
but i get error like /bin/sh: 1: ./pocketbase: not found

this is my dockerfile 
{FROM ubuntu
COPY . /app
WORKDIR /app
ENV PORT 8000
CMD [ "./pocketbase","serve","--http=0.0.0.0:$PORT","--debug" ]
EXPOSE 8000
}

and its just getin error
i read some other discussion with same issue and i try everything but it dosent work for me
please what is wrong with my dockerfile?

**Comment by tonyhart7** [comment]
You can just git clone the pocketbase repo on empty ubuntu image and build from it, dnt need docker image for it

**Reply by triborg34** [reply]
> You can just git clone the pocketbase repo on empty ubuntu image and build from it, dnt need docker image for it

How i can do that?
i dont know how to this

**Reply by kingdevnl** [reply]
You should Not host this on heroku. the database will reset every 10 minuts

**Reply by triborg34** [reply]
> You should Not host this on heroku. the database will reset every 10 minuts

i want to try fly.io
but also need docker to deploy

**Reply by tonyhart7** [reply]
> > You can just git clone the pocketbase repo on empty ubuntu image and build from it, dnt need docker image for it
> 
> How i can do that? i dont know how to this

try Railway.app

you can git clone this repo and just add dockerfile inside it for build and run pocketbase

**Comment by ganigeorgiev** [comment]
@triborg34 As others mentioned, PocketBase cannot be hosted on Heroku beacause it doesn't offer persistent storage.

Fly.io or any other traditional VPS service will be more suitable.

About the Dockerfile - we don't have official Docker image yet, but you can check the community examples from here - https://github.com/pocketbase/pocketbase/issues/92.

**Reply by triborg34** [reply]
@ganigeorgiev

ok i go to fly.io
and can i use that dockerfile u mention?

Or someone can teach me how deploy to fly.io

{its practice app so i want a free host to practice}

**Comment by tigawanna** [comment]
Thank you guys for this great product. 
on this topic is it hostable on something like the GCP app engine since it has persistance

**Reply by ganigeorgiev** [reply]
Yes, as long as you mount a persistent volume, it should work just fine.

**Comment by ganigeorgiev** [comment, marked as answer]
@triborg34 You could find a sample guide how to deploy on fly.io here - https://github.com/pocketbase/pocketbase/discussions/537.
